Beam/Include/Beam/Threading/Sync.hpp

```diff
@@ -168,7 +264,7 @@ namespace Threading {
 
   template<typename T, typename MutexType>
   Sync<T, MutexType>::Sync(Sync& sync)
-      : Sync{static_cast<const Sync&>(sync)} {}
+    : Sync{static_cast<const Sync&>(sync)} {}
```

Comment:
We no longer use braces for initialization, please update to use regular parenthesis (```Sync(...)```).